### PR TITLE
chore: register a dummy Read Receipts model in CE

### PR DESCRIPTION
**Pull request by pierre-lehnen-rc**
so that the model is always defined even when the EE code is not executed

## Issue(s)

FOSS-9

### CODE REVIEW

File: apps/meteor/server/models/dummy/ReadReceipts.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	constructor() {
+		super('read_receipts');
+	}
+
+	findByMessageId(_messageId: string): FindCursor<ReadReceipt> {
+		return this.find({});
+	}
+
+	removeByUserId(_userId: string): Promise<DeleteResult> {
+		return this.deleteMany({});
```
Comment (sampaiodiego) [inline apps/meteor/server/models/dummy/ReadReceipts.ts:17]: this is scary  🙈 

my understanding is that dummy models will never be actually called, so should them only throw errors? having this operations here are way too scary

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

File: apps/meteor/server/models/dummy/ReadReceipts.ts (hunk 1 above)
Comment (pierre-lehnen-rc) [reply apps/meteor/server/models/dummy/ReadReceipts.ts:17]: it's not an operation, it's just calling the mocked `deleteMany` that returns a compatible `DeleteResult` without doing anything.

Comment (pierre-lehnen-rc) [reply apps/meteor/server/models/dummy/ReadReceipts.ts:17]: The dummy model doesn't even have a mongo connection to operate on.

Comment (sampaiodiego) [reply apps/meteor/server/models/dummy/ReadReceipts.ts:17]: lol yeah I missed that 😛 my bad.

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**